www/install/createTables.sql

```diff
@@ -2050,7 +2051,7 @@ CREATE TABLE `traps` (
   `traps_advanced_treatment_default` enum('0','1', '2') DEFAULT '0',
   `traps_timeout` int(11) DEFAULT NULL,
   `traps_exec_interval` int(11) DEFAULT NULL,
-  `traps_exec_interval_type` enum('0','1', '2') DEFAULT '0',
+  `traps_exec_interval_type` enum('0','1','2','3') DEFAULT '0',
```

Comment:
missing documentation in your PR. At what correspond the type 3 ?

Reply:
I'm waiting the merge before doing the doc

Reply:
Why wait the merge and make another PR when this is linked and you may forgot ?